.travis.yml

```diff
@@ -3,6 +3,7 @@ branches:
   - master
 language: python
 python:
+  - "3.7-dev"
```

Comment:
Which version of Python do you get with this? FWIW I think it should be possible to use latest release 3.7, I did recently for another project, see [this .travis.yml](https://github.com/zarr-developers/numcodecs/blob/master/.travis.yml).